src/m365/entra/commands/app/app-set.ts

```diff
@@ -118,11 +125,16 @@ class EntraAppSetCommand extends GraphCommand {
     this.optionSets.push({ options: ['appId', 'objectId', 'name'] });
   }
 
+  #initTypes(): void {
+    this.types.boolean.push('allowPublicClientFlows');
+  }
+
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
     try {
       let objectId = await this.getAppObjectId(args, logger);
       objectId = await this.configureUri(args, objectId, logger);
       objectId = await this.configureRedirectUris(args, objectId, logger);
+      objectId = await this.updateAllowPublicClientFlows(args, objectId, logger);
```

Comment:
I do feel it's not really necessary to have three functions that all execute a patch request on the same app. To refactor it all is a bit too much for now, but at least we could combine the configureUri function with the updatePublicClientFlows function in a single updateApp function.

What do you think?

Reply:
Ok, good point. I still have a #1963 task where I would touch on these commands more deeply. Therefore, I can resolve this suggestion there